apps/ledger-live-desktop/src/renderer/families/evm/SendAmountFields/GasPriceField.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useCallback } from "react";
+import { BigNumber } from "bignumber.js";
+import invariant from "invariant";
+import { Account } from "@ledgerhq/types-live";
+import { Transaction, TransactionStatus } from "@ledgerhq/coin-evm/types";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import FeeSliderField from "~/renderer/components/FeeSliderField";
+import { Range, inferDynamicRange } from "@ledgerhq/live-common/range";
+type Props = {
+  account: Account;
+  transaction: Transaction;
+  status: TransactionStatus;
+  updateTransaction: (updater: (_: Transaction) => Transaction) => void;
+};
+const fallbackGasPrice = inferDynamicRange(BigNumber(10e9));
+let lastNetworkGasPrice: Range | undefined; // local cache of last value to prevent extra blinks
+
+const FeesField = ({ account, transaction, status, updateTransaction }: Props) => {
+  invariant(transaction.family === "evm", "FeeField: evm family expected");
+  const bridge = getAccountBridge(account);
+  const onGasPriceChange = useCallback(
+    gasPrice => {
+      updateTransaction(transaction =>
+        bridge.updateTransaction(transaction, {
+          gasPrice,
+          feesStrategy: "custom",
+        }),
+      );
+    },
+    [updateTransaction, bridge],
+  );
+
+  const { gasOptions } = transaction;
+
+  // FIXME: gasPrice should always be defined if gasOptions is defined
```

Comment:
Why the need for a FIXME here ? 
We're creating thing from scratch here, we can't allow any FIXME/TODO imo

Reply:
Indeed, no need for a FIXME here, forgot to remove that after I finished the implementation.
But I still added an `invariant` on `gasOptions` for good mesures since `gasOptions` should always be defined at this stage, even though according to TS, `transaction. gasOptions` is of type `GasOptions | undefined`